examples/scripts/vsft_llava.py

```diff
@@ -27,6 +27,9 @@
     --use_peft \
     --dataloader_num_workers 32 \
     --lora_target_modules=all-linear
+
+For LLaVA-NeXT, use: (requires transformers>=4.45)
```

Comment:
@lewtun @kashif would this be enough for the version?

Reply:
This is fine for me - if we get more issues we can always add a stronger check in the trainer